packages/blocks/README.md

```diff
@@ -665,6 +665,10 @@ _Returns_
 
 -   `Array|string`: A list of blocks or a string, depending on `handlerMode`.
 
+### PATHS_WITH_MERGE
+
+Undocumented declaration.
```

Comment:
I'm not 100% sure what this is used for!

Reply:
If the variable name is `__EXPERIMENTAL_PATHS_TO_MERGE` the doc generator will skip it. It's best if it's experimental at the moment. We may be able to do this differently later.

Reply:
Done, thank you!